pkg/providers/apisix/apisix_cluster_config.go

```diff
@@ -219,15 +219,6 @@ func (c *apisixClusterConfigController) handleSyncErr(obj interface{}, err error
 		c.MetricsCollector.IncrSyncOperation("clusterConfig", "success")
 		return
 	}
-	event := obj.(*types.Event)
```

Comment:
refs: https://github.com/apache/apisix-ingress-controller/issues/810

1. There may be an expired event during the retry, which means it was deleted when the retry occurred, and this event will continue to be triggered.
2. What is the reason why this resource status needs to be retried? What should be the retry strategy? I think rough deletion is not advisable.
3. The changes here are not related to the title of the PR. If you think you need to try again, please submit a separate PR for discussion